test/hotspot/jtreg/runtime/cds/appcds/SharedRegionAlignmentTest.java

```diff
@@ -68,7 +72,7 @@ static void testCombo() throws Exception {
                                                  TestCommon.list(mainClass),
                                                  TestCommon.concat(dumpLP, logArg));
             out.shouldContain("Dumping shared data to file");
-            boolean is_aligned_64k = out.getStdout().contains(logFor64K);
+            boolean is_alignment_logged = out.getStdout().contains(regionAlignmentString + regionAlignment);
```

Comment:
Is `is_alignment_logged` still needed? Now both dump time and run time should contain the `regionAlignmentString + regionAlignment` string. Also, I think it's better to use a local variable like:

```
String expectedAlignment = regionAlignmentString + regionAlignment;
```

Reply:
Yes, it should be contained in both dump/runtime time, will remove it. Thanks.